### PR TITLE
Finish the NEWS sumup

### PR DESCRIPTION
**Pull request by cedlemo**
I have added a global section because I didn't know where those changes should be added.

It takes me a lot of time. Maybe we should do this more frequently?

### CODE REVIEW

File: NEWS [hunk 1]
```diff
@@ -91,7 +261,15 @@ It improves GTK+ 3 support release. Or something...
...
   * Christopher L. Ramsey
-  * ...
+  * Hiroyuki ITo
+  * dutchhome Jon Raiford
+  * Rob Brackett
+  * Abby Archer
+  * Yuuki Harano
+  * detlef
+  * Aurélien Jacobs
+  * Hirochi Hatake
```
Comment (cosmo0920) [inline NEWS]: It's sad for my name typo.... :cry: 

```diff
-Hirochi
+Hiroshi
```

Comment (cedlemo) [inline NEWS]: Oh I really embarassed for this! 
ごめんなさい!! (I hope this is the good way to say that I am sorry)

Comment (cosmo0920) [inline NEWS]: Don't worry, we just fix it when someone find wrong things! :relaxed:

**Comment by kou** [comment]
Thanks!!!

**Comment by cosmo0920** [comment]
:blush:

**Comment by kou** [comment]
> I have added a global section because I didn't know where those changes should be added.

We're using "All" section for it. I changed the section name.

**Comment by kou** [comment]
> It takes me a lot of time. Maybe we should do this more frequently?

If it reduces resources for a new release, I agree with it.

**Pull request merged**